doc/articles/getting-started/GettingStartedWithTests.md

```diff
@@ -0,0 +1,61 @@
+---
+uid: Uno.Authoring.Tests
+---
+# How-To: Creating an application with tests using Uno.Extensions
+ 
+This tutorial will walk through how to create an Uno application with the `dotnet new` tool, which is already configured to use the Uno.Extensions.
+
+> [!NOTE]
+> Make sure to setup your environment first by [following our instructions](xref:Uno.GetStarted.vs2022).
+  
+## 1. Exploring the Solution
+ 
+The generated solution will contain:
+ 
+* *MyProjectName* - for application logic, and other constructs like view models and services, as well as the pages, controls, and other views that make up the UI of the application.
+* *MyProjectName/Platforms* - platform-specific folders for each supported platform.
+* *MyProjectName.Tests* and *MyProjectName.UI.Tests* - for writing unit and UI tests respectively.
+* *MyProjectName.Server* - backend server projects.
```

Comment:
The server project is not included in either blank or recommended template any more, so should be removed from here